src/core/views/popup.js

```diff
@@ -39,6 +39,10 @@ const BaseWrapper = styled.div`
   position: ${p => (p.isContained ? 'absolute' : 'fixed')};
   max-width: 100%;
   z-index: 10001;
+  @media screen and (min-width: 1024px) {
```

Comment:
Did you test this locally? This does not seem to fix the issue with centering it. The min-width and min-height are specially needed for small screens (under 1024).

Reply:
I think you need to change the logic in `popupWrapper` below where `top` and `left` are calculated.

Reply:
🤔  I see it centered, also the ticket talks about desktop not mobile 😕

Reply:
Did you check if it is really centered?

![Screenshot 2020-11-23 at 16 00 38](https://user-images.githubusercontent.com/1561771/99978090-cef5c880-2da5-11eb-9aa9-315758c4b3a5.png)

Since we do not determine mobile based on screen size, I think the popup should keep minimum size on desktop regardless of screen size - that means even for screens < 1024px wide.

Reply:
Are we looking at different embeds? 
![Screenshot 2020-11-23 at 16 13 53](https://user-images.githubusercontent.com/1560118/99979123-1597f280-2da7-11eb-8183-6e93d147b082.png)

Reply:
> I think the popup should keep minimum size on desktop regardless of screen size - that means even for screens < 1024px wide

Sorry mate, this line is quite confusing to me,  we talk about desktop then < than 1024px 😄  shall we jump in a chat?

Reply:
I would remove the media query and I think its good to go. With media query this is what it looks like on desktop when you resize browser window: 
![Screenshot 2020-11-24 at 17 15 11](https://user-images.githubusercontent.com/1561771/100120835-b1ddfa00-2e78-11eb-833a-401ddec13691.png)

Reply:
You're right, I messed up when rebasing from master (I had conflicts)